packages/integrations/src/transactionactivity.ts

```diff
@@ -93,6 +95,10 @@ export class TransactionActivity implements Integration {
     if (TransactionActivity._enabled !== undefined) {
       return TransactionActivity._enabled;
     }
+    // This happens only in test cases where the integration isn't initalized properly
+    if (!TransactionActivity.options || isNaN(TransactionActivity.options.tracesSampleRate)) {
```

Comment:
Remember that `isNaN('')` `isNaN([])` `isNaN(true)` `isNaN(false)` will also report `false`. You can wrap the input in `parseInt` to make it better, or use:
```
typeof TransactionActivity.options.tracesSampleRate !== 'number'
```